artifacts/pom.xml

```diff
@@ -0,0 +1,168 @@
+		<esignet-mock-wrapper.version>0.9.2-SNAPSHOT</esignet-mock-wrapper.version>
+		<esignet-mock-wrapper.fileName>esignet-mock-wrapper.jar</esignet-mock-wrapper.fileName>
+		
+		<!-- esignet IDA wrapper -->
+		<esignet-ida-wrapper.version>1.2.1-SNAPSHOT</esignet-ida-wrapper.version>
+		<esignet-ida-wrapper.fileName>esignet-ida-wrapper.jar</esignet-ida-wrapper.fileName>
+		<esignet-digital-credential-wrapper.version>0.1.0-SNAPSHOT</esignet-digital-credential-wrapper.version>
+                <esignet-digital-credential-wrapper.fileName>sunbird-rc-esignet-integration-impl.jar</esignet-digital-credential-wrapper.fileName>
+		
+		<esignet-plugins.location>/usr/share/nginx/html/artifactory/libs-release-local/esignet/esignet-plugins</esignet-plugins.location>
+		<esignet-mock-plugin.version>1.3.0-SNAPSHOT</esignet-mock-plugin.version>
+		<esignet-mock-plugin.fileName>esignet-mock-plugin.jar</esignet-mock-plugin.fileName>
+		<mosip-identity-plugin.version>1.3.0-SNAPSHOT</mosip-identity-plugin.version>
+		<mosip-identity-plugin.fileName>mosip-identity-plugin.jar</mosip-identity-plugin.fileName>
+		
+	</properties>
+	
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+        			<version>3.5.0</version>
+        <executions>
+          <execution>
+            <id>copy</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+	  </execution>
+	</executions>
+			<configuration>
+				<artifactItems>
+
+<!-- artifactItem section for esignet authentication wrapper -->
+				<artifactItem>
+					<groupId>io.mosip.esignet.mock</groupId>
+					<artifactId>mock-esignet-integration-impl</artifactId>
+					<version>${esignet-mock-wrapper.version}</version>
+					<outputDirectory>${esignet-wrapper.location}</outputDirectory>
+					<destFileName>${esignet-mock-wrapper.fileName}</destFileName>
+					<type>jar</type>
+				</artifactItem>
+					<artifactItem>
+						<groupId>io.mosip.authentication</groupId>
+						<artifactId>esignet-integration-impl</artifactId>
+						<version>${esignet-ida-wrapper.version}</version>
+						<outputDirectory>${esignet-wrapper.location}</outputDirectory>
+						<destFileName>${esignet-ida-wrapper.fileName}</destFileName>
+						<type>jar</type>
+					</artifactItem>
+					<artifactItem>
+                                                <groupId>io.mosip.esignet.sunbirdrc</groupId>
+	                                        <artifactId>sunbird-rc-esignet-integration-impl</artifactId>
+						<version>${esignet-digital-credential-wrapper.version}</version>
+						<outputDirectory>${esignet-wrapper.location}</outputDirectory>
+						<destFileName>${esignet-digital-credential-wrapper.fileName}</destFileName>
+						<type>jar</type>
+					</artifactItem>
+<!-- artifactItem section for certify plugin -->				
+					<artifactItem>
```

Comment:
Let's remove this artifact as it is part of inji-certify, Once the 1.5.0 version compatible Sunbird plugin is available we can add it back.